nextflow.config

```diff
@@ -185,7 +186,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base'
+    validationSchemaIgnoreParams     = 'genome,genomes,igenomes_base'
     validationShowHiddenParams       = false
     validate_params                  = true
```

Comment:
This should set to false according to https://nf-co.re/docs/contributing/pipeline_release_review_guidelines

Reply:
Nevermind, it's from the nf-core template